src/App.vue

```diff
@@ -19,12 +19,12 @@
           <div class="col-span-12">
             <div class="text-sm">{{slotProps.message.detail}}</div>
             <div class="mt-1 text-xs">{{slotProps.message.detail2}}</div>
-            <div :style="`font-size: ${slotProps.message.detail3RemSize ? slotProps.message.detail3RemSize: 0.5}rem`" v-if="slotProps.message.url">
+            <div :style="`font-size: ${slotProps.message.detail3RemSize ? slotProps.message.detail3RemSize: 0.6}rem`" v-if="slotProps.message.url">
```

Comment:
sizing up at the toast message will cause the 64 characters to overflow outside the message box, should define the detail3RemSize when not using the default

Reply:
issue #656 created.